Xamarin.PropertyEditing.Mac/Controls/Custom/NumericSpinEditor.cs

```diff
@@ -0,0 +1,287 @@
+﻿using System;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class NumericSpinEditor : NSView
+	{
+		protected NumericTextField numericEditor;
+		protected NSStepper stepper;
```

Comment:
These should not be exposed outside of this class. The values stored in them must be kept in sync otherwise Bad Things happen. If we publicly expose them we can't guarantee they'll always be in sync